yarn.lock

```diff
@@ -12407,14 +12407,6 @@ react-test-renderer@^18.2.0:
     react-shallow-renderer "^16.15.0"
     scheduler "^0.23.2"
 
-react-tooltip@4.2.14:
-  version "4.2.14"
-  resolved "https://registry.yarnpkg.com/react-tooltip/-/react-tooltip-4.2.14.tgz#8e06b5926fdf6672e78d8ccadaa16bef40d131d7"
-  integrity sha512-hS2kAlpjyH5MXL9DaGKsdmEFCIEuMD2RZXkEJeNjmDe05dHpqj93o5JgpmczAgQFk099+JSsnHUDo7pIOuyMDQ==
-  dependencies:
-    prop-types "^15.7.2"
-    uuid "^7.0.3"
-
 react-tooltip@^4.2.21:
```

Comment:
Just checking, is this one still needed?

Reply:
It's a dependency of `@theia/core` - Indeed, I do not see directly used in this repo here, but it might be used in the Theia examples in this repo.

```
$ yarn why react-tooltip
yarn why v1.22.22
[1/4] Why do we have the module "react-tooltip"...?
[2/4] Initialising dependency graph...
[3/4] Finding dependency...
[4/4] Calculating file sizes...
=> Found "react-tooltip@4.5.1"
info Reasons this module exists
   - "_project_#browser-theia-trace-example-app#@theia#core" depends on it
   - Hoisted from "_project_#browser-theia-trace-example-app#@theia#core#react-tooltip"
info Disk size without dependencies: "956KB"
info Disk size with unique dependencies: "1.48MB"
info Disk size with transitive dependencies: "1.6MB"
info Number of shared dependencies: 6
```

Reply:
I did check this out before submitting this PR.  I made sure react-tooltip was removed from all of the codebase as well as all `package.json` files.  I came to the same conclusion as @marcdumais-work, that `@theia` was using it somewhere.

Reply:
In any case, this dependency will not be part of the published npm packages (since they do not depend on it), so we should be good.